bzt/bza.py

```diff
@@ -741,20 +745,24 @@ def get_addr(self):
 
         proxy_info = response['result']
         if proxy_info:
+            self.log.info('Proxy already exists')
+            if 'username' in proxy_info:
+                msg = 'Proxy has auth, unable to use it'
+                self.log.info(msg)
+                raise BZAProxyException(msg)
+
             self.log.info('Using existing recording proxy...')
             if proxy_info['status'] == 'active':
                 self.log.info('Proxy is active, stop it')
                 self.stop()
         else:
             self.log.info('Creating new recording proxy...')
-            response = self._request(self.address + '/api/latest/proxy', method='POST')
+            response = self._request(self.address + '/api/latest/proxy', method='POST', data={'auth': False})
             proxy_info = response['result']
 
         self._request(self.address + '/api/latest/proxy/recording/clear', method='POST')
 
-        return '%s:%s@%s:%s' % (
-            proxy_info['username'],
-            proxy_info['password'],
+        return 'http://%s:%s' % (
```

Comment:
Did you figure out if it finally matters or not to have http://?

Reply:
It actually doesn't matter, I'll remove it

Reply:
let's keep it for now